ci-operator/config/openshift/installer/openshift-installer-master__terraform.yaml

```diff
@@ -0,0 +1,110 @@
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.18-openshift-4.12
+  ocp_builder_rhel-8-golang-1.19-openshift-4.14:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.19-openshift-4.14
+  ocp_builder_rhel-8-golang-1.20-openshift-4.14:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |
+    FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14 AS macbuilder
+    ARG TAGS=""
+    WORKDIR /go/src/github.com/openshift/installer
+    COPY . .
+    RUN GOOS=darwin GOARCH=amd64 DEFAULT_ARCH="$(go env GOHOSTARCH)" make -C terraform all
+    RUN find /go/src/github.com/openshift/installer/terraform/bin -executable ! -name "terraform" -type f -exec rm -f {} \;
+
+    FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14 AS macarmbuilder
+    ARG TAGS=""
+    WORKDIR /go/src/github.com/openshift/installer
+    COPY . .
+    RUN CGO_ENABLED=0 GOOS=darwin GOARCH=arm64 DEFAULT_ARCH="$(go env GOHOSTARCH)" make -C terraform all
+    RUN find /go/src/github.com/openshift/installer/terraform/bin -executable ! -name "terraform" -type f -exec rm -f {} \;
+
+    FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14 AS linuxbuilder
+    ARG TAGS=""
+    WORKDIR /go/src/github.com/openshift/installer
+    COPY . .
+    RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 DEFAULT_ARCH="$(go env GOHOSTARCH)" make -C terraform all
+    RUN find /go/src/github.com/openshift/installer/terraform/bin -executable ! -name "terraform" -type f -exec rm -f {} \;
+
+    FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14 AS linuxarmbuilder
+    ARG TAGS=""
+    WORKDIR /go/src/github.com/openshift/installer
+    COPY . .
+    RUN CGO_ENABLED=0 GOOS=linux GOARCH=arm64 DEFAULT_ARCH="$(go env GOHOSTARCH)" make -C terraform all
+    RUN find /go/src/github.com/openshift/installer/terraform/bin -executable ! -name "terraform" -type f -exec rm -f {} \;
+
+    FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14 AS builder
+    WORKDIR /go/src/github.com/openshift/installer
+    COPY --from=macbuilder /go/src/github.com/openshift/installer/terraform/bin/ terraform/bin/
+    COPY --from=macarmbuilder /go/src/github.com/openshift/installer/terraform/bin/ terraform/bin/
+    COPY --from=linuxbuilder /go/src/github.com/openshift/installer/terraform/bin/ terraform/bin/
+    COPY --from=linuxarmbuilder /go/src/github.com/openshift/installer/terraform/bin/ terraform/bin/
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.18-openshift-4.12:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12
+    ocp_builder_rhel-8-golang-1.19-openshift-4.14:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.14
+    ocp_builder_rhel-8-golang-1.20-openshift-4.14:
```

Comment:
Looks like only this one is really required (1.18-4.12 and 1.19-4.14 are not referenced in the dockerfile)?

Reply:
Removed.